patches/minecraft/net/minecraft/tileentity/TileEntityBrewingStand.java.patch

```diff
@@ -1,15 +1,54 @@
 --- ../src-base/minecraft/net/minecraft/tileentity/TileEntityBrewingStand.java
 +++ ../src-work/minecraft/net/minecraft/tileentity/TileEntityBrewingStand.java
-@@ -118,7 +118,7 @@
+@@ -104,84 +104,21 @@
  
-                 for (int i = 0; i < 3; ++i)
-                 {
+     private boolean func_145934_k()
+     {
+-        if (this.field_145945_j[3] != null && this.field_145945_j[3].field_77994_a > 0)
```

Comment:
This patch can be made a lot smaller by not deleting the code.
Is this the code that is int he default canBrew implementation?
If so it may be simpler to make this into a static function stub and have the default implementation call it.
Using comments of course to state that this is what you're doing and that this function shouldn't be called elsewise.